src/main/java/io/lettuce/core/AbstractRedisReactiveCommands.java

```diff
@@ -1635,6 +1636,16 @@ public Mono<Map<K, Long>> pubsubNumsub(K... channels) {
         return createMono(() -> commandBuilder.pubsubNumsub(channels));
     }
 
+    @Override
+    public Flux<K> pubsubShardChannels() {
+        return createDissolvingFlux(() -> commandBuilder.pubsubShardChannels());
```

Comment:
note (not important) : could use the short syntax here, similar to line 1620

Reply:
replaced with method reference